appveyor.yml

```diff
@@ -43,6 +38,14 @@ environment:
       CONDA_PY: 35
       CONDA_INSTALL_LOCN: C:\\Miniconda35-x64
 
+    - TARGET_ARCH: x86
+      CONDA_PY: 36
+      CONDA_INSTALL_LOCN: C:\\Miniconda
```

Comment:
Can you skip py36 builds on windows and rerender? py35 and py36 both use vc14 and therefore there's no need to build twice.